cli/go.mod

```diff
@@ -0,0 +1,83 @@
+module github.com/oasisprotocol/oasis-sdk/cli
+
+go 1.17
+
+replace github.com/oasisprotocol/oasis-sdk/client-sdk/go => ../client-sdk/go
+
+require (
+	github.com/AlecAivazis/survey/v2 v2.3.2
+	github.com/adrg/xdg v0.4.0
+	github.com/mitchellh/mapstructure v1.4.2
+	github.com/oasisprotocol/curve25519-voi v0.0.0-20210716083614-f38f8e8b0b84
+	github.com/oasisprotocol/deoxysii v0.0.0-20200527154044-851aec403956
+	github.com/oasisprotocol/oasis-core/go v0.2103.6
+	github.com/oasisprotocol/oasis-sdk/client-sdk/go v0.1.0
+	github.com/olekukonko/tablewriter v0.0.5
+	github.com/shopspring/decimal v1.3.1
+	github.com/spf13/cobra v1.2.1
```

Comment:
Could you update cobra to 1.3.0? It has some completion sub-command improvements.

Reply:
Done.